controllers/usersignup/usersignup_controller.go

```diff
@@ -655,7 +657,14 @@ func (r *Reconciler) ensureNewMurIfApproved(reqLogger logr.Logger, config toolch
 	}
 
 	// Provision the MasterUserRecord
-	return r.provisionMasterUserRecord(reqLogger, config, userSignup, targetCluster, userTier)
+	if err := r.provisionMasterUserRecord(reqLogger, config, userSignup, targetCluster, userTier); err != nil {
+		return err
+	}
+	// track activation in Segment
+	if r.SegmentClient != nil {
+		return r.SegmentClient.TrackAccountActivation(userSignup.Spec.Username)
```

Comment:
@ibuziuk @alexeykazakov I would need to get confirmation that `userSignup.Spec.Username` is the field to use here, so it matches with what CRW and Web Console send.

Reply:
`userSignup.Spec.Username` is the user's sso username. My understanding it's what CRW and WebConsole use as well.

Reply:
ok, thanks for confirming!